src/libcore/time.rs

```diff
@@ -501,6 +524,15 @@ impl Mul<u32> for Duration {
     }
 }
 
+#[unstable(feature = "time_units", issue = "0")]
+impl Mul<Duration> for u32 {
```

Comment:
This impl is going to be insta-stable unfortunately.

Reply:
Should I keep the attribute?

Reply:
@newpavlov  The latter.

Reply:
Hm, what should I use for `feature` then? With `time_units` it [can not be built](https://github.com/rust-lang/rust/pull/52556#issuecomment-406625862).